### ISSUE TITLE
[BUG] nvim_win_set_hl_ns is a nil value

### ISSUE THREAD

**Issue by notseanray** [uga-rosa/ccc.nvim#12]
When attempting to open the color picking menu with `:CccPick` it creates an error like below. After hitting enter it opens the menu normally but without color in the selection window.

![image](https://user-images.githubusercontent.com/61510083/190869904-8fa4e156-dbd4-4488-b90e-a4ead7c32bd0.png)
![image](https://user-images.githubusercontent.com/61510083/190869992-8dadf899-a8a8-4852-aa95-1fe7fedfb86b.png)

The problem seems to originate [here](https://github.com/uga-rosa/ccc.nvim/blob/main/lua/ccc/ui.lua#L69). The function obviously exist according to [this](http://neovim.io/doc/user/api.html).

I am using branch 0.7.2 of this plugin.

`nvim --version`
```
NVIM v0.7.2
Build type: RelWithDebInfo
LuaJIT 2.1.0-beta3
Compilation: /usr/bin/x86_64-pc-linux-gnu-gcc -march=znver1 -mtune=znver1 -maes -fstack-protector -O2 -pipe -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=1 -DNVIM_TS_HAS_SET_MATCH_LIMIT -DNVIM_TS_HAS_SET_ALLOCATOR -Og -g -Wall -Wextra -pedantic -Wno-unused-parameter -Wstrict-prototypes -std=gnu99 -Wshadow -Wconversion -Wdouble-promotion -Wmissing-noreturn -Wmissing-format-attribute -Wmissing-prototypes -Wimplicit-fallthrough -Wsuggest-attribute=pure -Wsuggest-attribute=const -Wsuggest-attribute=malloc -Wsuggest-attribute=cold -Wvla -fstack-protector-strong -fno-common -fdiagnostics-color=always -DINCLUDE_GENERATED_DECLARATIONS -D_GNU_SOURCE -DNVIM_MSGPACK_HAS_FLOAT32 -DNVIM_UNIBI_HAS_VAR_FROM -DMIN_LOG_LEVEL=3 -I/var/tmp/portage/app-editors/neovim-0.7.2/work/neovim-0.7.2_build/config -I/var/tmp/portage/app-editors/neovim-0.7.2/work/neovim-0.7.2/src -I/usr/include -I/var/tmp/portage/app-editors/neovim-0.7.2/work/neovim-0.7.2_build/src/nvim/auto -I/var/tmp/portage/app-editors/neovim-0.7.2/work/neovim-0.7.2_build/include
Compiled by portage@localhost

Features: +acl +iconv +tui
See ":help feature-compile"

   system vimrc file: "/etc/vim/sysinit.vim"
  fall-back for $VIM: "/usr/share/nvim"

Run :checkhealth for more info
```

`uname -a`
```
Linux Gentoo 5.18.1-gentoo-r2 #3 SMP PREEMPT_DYNAMIC Sat Jun 18 19:33:04 EDT 2022 x86_64 AMD Ryzen 7 1700 Eight-Core Processor AuthenticAMD GNU/Linux
```

I did not see any other issues that mentioned this. Thank you!

**Comment by uga-rosa** [comment]
Same as #10.
As I noted in README, please use branch `0.7.2`.

**Comment by notseanray** [comment]
> Same as #10. As I noted in README, please use branch `0.7.2`.

I am using branch `0.7.2` with `Plug.vim`
`Plug 'uga-rosa/ccc.nvim', {'branch': '0.7.2'}`

The issue also states this
> I am using branch 0.7.2 of this plugin.

I removed the `ccc.nvim` folder in `nvim/plugged` and re-ran `:PlugInstall` that fixed it, guess it cloned the wrong branch at first and did not overwrite the folder. My apologies.

**Comment by uga-rosa** [comment]
I'm not using that API on that branch. Sorry for replying without reading much.

**Comment by notseanray** [comment]
No worries, in the end it was my mistake regardless and that is a perfectly valid assumption. Thank you for this plugin it is wonderful!

**Comment by notseanray** [comment]
If you are using [Plug](https://github.com/junegunn/vim-plug) or [Packer](https://github.com/wbthomason/packer.nvim) you can specify the commit to use. Testing with older commits may be worthwhile to help fix this issue. Be sure to delete previous plugin code when testing with different versions as that was the reason it wasn't working for me initially.

**Comment by nikbrunner** [comment]
Yeah, I am using Packer and I pinned a commit. Here is what I was using: 

```lua
    use({
        "uga-rosa/ccc.nvim",
        commit = "08abe425363279ff8a80cd237ac0ea003bee4863",
    })
```

I wanted to update today and also tried the new branch, but then it was broken again.
When I tried to reinstall the commit I mentioned above, it seems that the commit is no longer existent.

I now tried a reinstallation with these commits, which are both generally in the time area of my pinned commit, but to no success. I always get `nvim_win_set_hl_ns is a nil value`. 

- https://github.com/uga-rosa/ccc.nvim/commit/9856dd631ecb80355dd936f3b9cbd4bea6040014
- https://github.com/uga-rosa/ccc.nvim/commit/832229fe18ce2395634a97fb53d9ea1f717e0195
